Java/JDI/jdi-uitest-web/src/main/java/com/epam/jdi/uitests/web/selenium/elements/complex/ComboBox.java

```diff
@@ -67,6 +74,73 @@ public String label() {
         return label.getText();
     }
 
+    public static void setUp(BaseElement el, Field field) {
+        if (!fieldHasAnnotation(field, JComboBox.class, IComboBox.class))
+            return;
+        ((ComboBox) el).setUp(field.getAnnotation(JComboBox.class));
+    }
+
+    public ComboBox<TEnum> setUp(JComboBox jComboBox) {
+        By root = findByToBy(jComboBox.root());
+        By value = findByToBy(jComboBox.value());
+        By list = findByToBy(jComboBox.list());
+        By expand = findByToBy(jComboBox.expand());
+        By textField = findByToBy(jComboBox.textField());
+        By labelLocator = findByToBy(jComboBox.labelLocator());
+
+
+        if (root == null) {
+            root = findByToBy(jComboBox.jRoot());
+        }
+
+        if (value == null) {
+            value = findByToBy(jComboBox.jValue());
+        }
+
+        if (list == null) {
+            list = findByToBy(jComboBox.jList());
+        }
+
+        if (expand == null) {
+            expand = findByToBy(jComboBox.jExpand());
+        }
+
+        if (textField == null) {
+            textField = findByToBy(jComboBox.jTextField());
+        }
+
+        if (labelLocator == null) {
+            labelLocator = findByToBy(jComboBox.jLabelLocator());
+        }
+
+        if (root != null) {
+            Element el = new Element(root);
+            el.setParent(getParent());
+            setParent(el);
+            setAvatar(root);
+        }
+
+        if (value != null) {
+            this.element = new GetElementType(value, this);
+            if (expander == null) this.expander = element;
+        }
+
+        if (list != null) {
+            this.allLabels = new GetElementType(list, this);
+        }
+
+        if (expand != null) {
+            this.expander = new GetElementType(expand, this);
+            if (element == null) this.element = expander;
```

Comment:
if statment should go on 1st line other stuff on subsequent lines + surround with curly braces.